doc/dev/mempool_accounting.rst

```diff
@@ -0,0 +1,55 @@
+What is a mempool?
+------------------
+A memory pool is a method for accounting the consumption of memory of
+a set of containers.
+Memory pools are statically declared (see pool_index_t), which serve
+the purpose of identifying memory related checks in bluestore.
+
+Some common mempools that we can track
+--------------------------------------
+
+- bloom_filter: Ceph uses bloom filters to account for tracking objects already
+  in the cache pool, in order to determine which objects in the pool are being
+  accessed.
+- bluestore_alloc: It accounts for actual allocations done by an allocator.
+- bluestore_inline_bl: measures encoded length of an inline buffer.
+- bluestore_fsck: file system consistency check for bluestore metadata, helps in
+  tracking and debugging during OSD repairs.
+- bluestore_txc: accounts for committed transactions counter.
+- bluestore_writing_deferred: measures those small writes which are deferred
+  i.e. first written into RocksDB WAL and later flushed to disk.
+- bluestore_writing: TBD
+- bluefs:  file-system-like interface that provides just enough functionality to
+  allow RocksDB to store its “files” and share the same raw device(s) with
+  BlueStore.
+- buffer_anon: stores arbitrary buffer data.
+- buffer_meta: all the metadata associated with buffer anon buffers.
+- bluestore_cache_data: mempool for writing and writing deferred.
+- bluestore_cache_onode: meta information of object node stored directed in
+  bluestore cache.
+- bluestore_cache_meta: key under PREFIX_OBJ where we are stored.
+- bluestore_cache_other: right now account's for
+
```

Comment:
I think the apostrophe is superfluous and the tail end of the description is missing

Reply:
hey @anthonyeleven this was intentional, as following being sub-points for bluestore_cache_other:
![image](https://user-images.githubusercontent.com/22215384/159728945-9484f2f9-8b28-40d3-b924-1514d28d9c7f.png)